### PR TITLE
[1LP][RFR] Make BZ references uniform

### PR DESCRIPTION
**Pull request by john-dupuy**
We have many places in tests where we reference BZs by different names. We should choose a uniform name for consistency in our code base. Since, `Bugzilla` seemed to be used the most, I chose that one. 

The scope of this PR is just to make the docblock BZ reference uniform. Later PRs will make use of a script to add meta data, such as `automates` to the test function. 

For reference the proper way to format BZs in the docblock going forward will be:

```python

def test_bugzilla(): 
    """
    Bugzilla:
        <bug_id1>
        <bug_id2>
        <bug_id3>
        ...
    """
    pass
```
Just the number (BZ ID) without a `*` or anything else preceding. I am working on a flake plugin to check this field. Each BZ id related to a test case should appear on its own line.

### CODE REVIEW

File: cfme/tests/automate/test_vmware_methods.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def test_vmware_vimapi_hotadd_disk(
             3. After the button is created, it goes to a VM's summary page, clicks the button.
             4. The test waits until the capacity of disks is raised.
 
-    Bugzillas:
+    Bugzilla:
         1211627, 1311221
```
Comment (valaparthvi) [inline cfme/tests/automate/test_vmware_methods.py]: Looking at Mike's PR #8675, you might want to keep one BZ on each line
```
Bugzilla:
        * 1211627
        * 1311221
````

Comment (john-dupuy) [reply cfme/tests/automate/test_vmware_methods.py]: Thanks! I think I missed a few of those `BZ:` refs on rebase. I like the pattern on Mike's PR so I'll change that too

Comment (john-dupuy) [reply cfme/tests/automate/test_vmware_methods.py]: Actually there are many such cases where `*` is absent, @mshriver do we want to include the `*` or leave it optional?

**Comment by john-dupuy** [comment]
Replaced BZ urls with the `Bugzilla: <id>` in test cases as well.

**Comment by john-dupuy** [comment]
Moving to WIP so we can wait for #8675 to be merged before this PR

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/automate/test_vmware_methods.py (hunk 1 above)
Comment (john-dupuy) [reply cfme/tests/automate/test_vmware_methods.py]: Discussing with Mike on chat, he suggested without the `*`.

**Comment by dajoRH** [comment]
Lint report for commit 0226bb7319b21862f83c742118d0c2db803fc334:

`cfme/tests/candu/test_host_graph.py`:
- :red_circle: Line 216:17: **F821** *undefined name 'datetime'*
- :red_circle: Line 216:34: **F821** *undefined name 'timedelta'*
- :red_circle: Line 256:5: **F821** *undefined name 'compare_data_with_unit'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 2]
```diff
@@ -749,7 +773,10 @@ def test_service_ansible_verbosity():
 @pytest.mark.tier(3)
 def test_service_ansible_playbook_cloud_credentials():
     """
-    https://bugzilla.redhat.com/show_bug.cgi?id=1444092When the service is
+    Bugzilla:
+        1444092
+
+    When the service is
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: Please in continue line...

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 3]
```diff
@@ -783,7 +810,10 @@ def test_service_ansible_playbook_order_credentials_usecredsfromservicedialog():
 @pytest.mark.tier(3)
 def test_service_ansible_playbook_machine_credentials_service_details_opsui():
     """
-    https://bugzilla.redhat.com/show_bug.cgi?id=1515561  When the service
+    Bugzilla:
+        1515561
+
+    When the service
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: same as above ^^

File: cfme/tests/cloud/test_providers.py [hunk 4]
```diff
@@ -516,10 +518,10 @@ def test_cloud_names_grid_floating_ips(appliance, ec2_provider, soft_assert):
 @pytest.mark.tier(3)
 def test_display_network_topology(appliance, openstack_provider):
     """
-        Bugzilla:
-            1343553
+    Bugzilla:
+        1343553
 
-        Steps to Reproduce:
+    Steps to Reproduce:
```
Comment (digitronik) [inline cfme/tests/cloud/test_providers.py]: Not in scope of PR but I would like to request you just move in polarion docstring as `testSteps`

File: cfme/tests/control/test_control_simulation.py [hunk 5]
```diff
@@ -25,8 +25,8 @@
 @pytest.mark.tier(1)
 def test_control_icons_simulation(appliance):
     """
-    Bugzillas:
-        * 1349147, 1690572
+    Bugzilla:
+        1349147, 1690572
```
Comment (digitronik) [inline cfme/tests/control/test_control_simulation.py]: Required:
line by line please

File: cfme/tests/test_manual.py [hunk 6]
```diff
@@ -7361,8 +7428,11 @@ def test_verify_orchestration_catalog_items_can_only_use_providers_that_are_visi
 @pytest.mark.tier(2)
 def test_osp_test_osp_volumes_are_cleaned_up_if_migration_fails_to_create_instance():
     """
-    V2V Test for https://bugzilla.redhat.com/show_bug.cgi?id=1651352
-    https://bugzilla.redhat.com/show_bug.cgi?id=1653412
+    V2V Test following BZs
+
+    Bugzilla:
+        1651352
+        * 1653412
```
Comment (digitronik) [inline cfme/tests/test_manual.py]: Required:
please remove this `*`

File: cfme/tests/test_manual.py [hunk 7]
```diff
@@ -7657,8 +7734,10 @@ def test_validate_cost_monthly_usage_network():
 def test_vm_tempate_ownership_nogroup():
     """
     test assigning no groups ownership for vm and templates
-    https://bugzilla.redhat.com/show_bug.cgi?id=1330022
-    https://bugzilla.redhat.com/show_bug.cgi?id=1456681
+    Bugzilla:
+        1330022
+        * 1456681
```
Comment (digitronik) [inline cfme/tests/test_manual.py]: remove this `*`

File: cfme/tests/test_manual.py [hunk 8]
```diff
@@ -8370,8 +8453,11 @@ def test_distributed_zone_create_duplicate():
 def test_edit_vm():
     """
     Edit infra vm and cloud instance
-    https://bugzilla.redhat.com/show_bug.cgi?id=1399141
-    https://bugzilla.redhat.com/show_bug.cgi?id=1399144
+
+    Bugzilla:
+        1399141
+        * 1399144
```
Comment (digitronik) [inline cfme/tests/test_manual.py]: same as above ^^

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 9]
```diff
@@ -833,7 +865,10 @@ def test_custom_button_order_ansible_playbook_service():
 @pytest.mark.tier(3)
 def test_service_ansible_overridden_extra_vars():
     """
-    https://bugzilla.redhat.com/show_bug.cgi?id=1444107Once a Ansible
+    Bugzilla:
+        1444107
+
+    Once a Ansible
     Playbook Service Dialog is built, it has default parameters, which can
```
Comment (digitronik) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: this also ^^

File: cfme/tests/webui/test_general_ui.py [hunk 10]
```diff
@@ -523,6 +527,6 @@ def test_custom_navigation_menu():
             1. Custom menu must be visible.
 
     Bugzilla:
-        * 1678151
+        1660460
```
Comment (mshriver) [inline cfme/tests/webui/test_general_ui.py]: Looks like this number changes? is this intentional?

**Comment by mshriver** [comment]
Looks good, spotted one number change that needs to be double checked.

File: cfme/tests/webui/test_general_ui.py (hunk 10 above)
Comment (john-dupuy) [reply cfme/tests/webui/test_general_ui.py]: Good catch, the original ID is the one we want!

Comment (digitronik) [reply cfme/tests/webui/test_general_ui.py]: @john-dupuy called `Eagles eyes` :) I missed this...

**Pull request merged**